package.json

```diff
@@ -85,8 +85,9 @@
       "tokenRef": "GITHUB_AUTH"
     }
   },
+  "packageManager": "pnpm@9.0.6",
```

Comment:
if you use pnpm@7 you won't need to remove node 14 from the matrix in this PR. It would be nice to have a newer pnpm but pnpm@7 with `auto-install-peers` false is good enough

Reply:
discussed in the meeting, ignore me!